static-site/src/sections/individual-group-page.jsx

```diff
@@ -0,0 +1,200 @@
+import React from "react";
+import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
+import styled from 'styled-components';
+import {
+  HugeSpacer,
+  FlexCenter
+} from "../layouts/generalComponents";
+import * as splashStyles from "../components/splash/styles";
+import DatasetSelect from "../components/Datasets/dataset-select";
+import GenericPage from "../layouts/generic-page";
+import { fetchAndParseJSON } from "../util/datasetsHelpers";
+import { StyledLink } from "../components/Datasets/list-datasets";
+
+function Title({avatarSrc, children}) {
```

Comment:
Refactoring recommendation, I would advocate pursuing one of  two styles here:

1. Make a number of styled components, with each of them doing one thing, and then call them all from the main render loop. E.g. `HeadingContainer`, `Title`, `Byline`, `GroupWebsite`, `Avatar`, `GroupDescription` etc. This is nice and declarative, and allows new developers who are reading the `render()` method for the first time to easily understand what's happening. 

2. Make an separate, higher level component such as `GroupHeading`, which takes in `this.props.sourceInfo` as its own prop and handles all the rendering. This can help comprehension of the render method if the solution in (1) becomes overly long. (Within this components file, you will most probably want to use the components described in (1).)

At the moment we've got a bit of both. I favour (2) as the complexity of this header section will only grow as we add in markdown parsing, but (1) would also work!